lib/src/office_js.dart

```diff
@@ -18,155 +18,14 @@ import 'package:js/js.dart';
 class Office {
   /// Ensures that the Office JS APIs are ready to be called by the add-in.
   external static Future<Info> onReady(dynamic Function(Info) callback);
-
-  /// The runtime environment of the add-in.
-  external static Context get context;
-
-  /// The result of an asynchronous call.
-  external static AsyncResultStatus get asyncResultStatus;
 }
 
 /// Input argument to [onReady()].
 @JS()
```

Comment:
what happened do you not need this class anymore?

Reply:
Those are some wrapper functions that I added when i thought i had to use the office dialog box (long long time ago). This class is not used very often, so I totally forgot to delete them :(